packages/paste-website/src/pages/components/tabs/index.mdx

```diff
@@ -85,20 +86,32 @@ export const getStaticProps = async () => {
 
 ### About Tabs
 
-The Tabs component allows a user to flip through multiple views within the same page.
+Tabs are labeled controls that allow users to switch between multiple views within a page. Tabs do not take the user to a new URL. To take users to a new URL, use [In Page Navigation](/components/in-page-navigation).
+
+Panels within a set of Tabs should present different views of similar, parallel information. For example, [these Twilio docs](https://www.twilio.com/docs/usage/api) contain a block of code on the right side of the page with Tabs, allowing users to view the code in different languages. [Reddit also uses tabs](https://www.reddit.com/t/technology/), switching the user’s view between **Posts**, **Communities**, and **Related Topics**. The Tabs in both of these examples control views; they don’t direct to a new page. Conversely, [the Github website](https://github.com/twilio-labs/paste) demonstrates how some interactions may seem like Tabs, but are actually in-page navigation. Clicking on one of the sections at the top, like **Code** or **Pull Requests**, changes the URL.
+
+Choose Tabs when users don’t need to see content from multiple views simultaneously. When thinking through what type of content to put in your Tabs, remember that the content should be parallel across the board. What lives in one Tab should be a different view of what lives in another Tab.
+
+_Source: [Nielsen Norman Group](https://www.nngroup.com/articles/tabs-used-right/)_
 
 ### Accessibility
 
 When the user is focused on a tab in a TabList, the following keyboard interactions apply:
 
-- Right and left arrow move the user between tabs and opens the currently focused tab's panel
-  - If the tabs are vertical, up and down arrows move between tabs instead of right and left
-  - If the currently focused tab is disabled, the tab panel does not open
+- Horizontal Tabs: `Right` and `Left` arrow keys move the user between Tabs and open the currently focused Tab's panel
+- Vertical Tabs: `Up` and `Down` arrow keys move between Tabs and open the currently focused Tab's panel
+- If the currently focused Tab is disabled, the Tab panel does not open
```

Comment:
suggestion: just remembered one thing that is worth calling out as a guideline for accessibility

- You must not place interactive elements inside a tab